source/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include "patch/iosu_kernel.h"
+
+#include <coreinit/thread.h>
+#include <coreinit/time.h>
+#include <coreinit/title.h>
+#include <sysapp/launch.h>
+#include <vpad/input.h>
+#include <sndcore2/core.h>
+
+#include "proc.hpp"
+#include <whb/log.h>
+#include <whb/log_udp.h>
+#include <whb/log_console.h>
+#include <whb/gfx.h>
+
+#define COLOR_CONSTANT 0x632A5CFF
+
+#define HBL_TITLE_ID (0x0005000013374842)
+#define MII_MAKER_JPN_TITLE_ID (0x000500101004A000)
+#define MII_MAKER_USA_TITLE_ID (0x000500101004A100)
+#define MII_MAKER_EUR_TITLE_ID (0x000500101004A200)
+
+int main(int argc, char* args[])
+{
+    ProcInit();
+    
+    WHBLogUdpInit();
+    WHBLogConsoleInit();
+
+    WHBLogConsoleSetColor(COLOR_CONSTANT);
+
+    WHBLogPrint("Welcome to the Nimble patcher, provided by the Pretendo team");
+    WHBLogPrint("The patcher will redirect Nintendo services to Pretendo");
+    WHBLogPrint("For Miiverse in game patches, you will need to run Inkay");
+    WHBLogPrint("via the WUPS loader.");
+    WHBLogPrint("\nPress A to install patches (NoSSL etc..)");
+    WHBLogPrint("\nPress B to exit");
+
+    WHBLogConsoleDraw();
+
+    VPADStatus status;
+    VPADReadError error = VPAD_READ_SUCCESS;
+
+    uint64_t titleID = OSGetTitleID();
+
+    bool isChannel = true;
+
+    if (titleID == HBL_TITLE_ID ||
+        titleID == MII_MAKER_JPN_TITLE_ID ||
+        titleID == MII_MAKER_USA_TITLE_ID ||
+        titleID == MII_MAKER_EUR_TITLE_ID)
+    {
+        isChannel = false;
+    }
+
+    if (isChannel)
+    {
+        WHBLogPrintf("Is channel");
+        AXInit(); 
+        // ^ I hate this, it's just an issue with the provided bootSound.btsnd in this repo. You can use another one and then repack via NUSPacker if you want.

```

Comment:
I hate this too! Is there a silent bootSound we can use instead, maybe?

Reply:
I’ll look around, if not I can just delete it or the silent bootSound idea